core/modules/admin_bar/admin_bar.inc

```diff
@@ -817,6 +868,11 @@ function admin_bar_theme_settings() {
   $form['components']['admin_bar.users']['#description'] = t('A count of the people currently logged into the site.');
   $form['components']['admin_bar.account']['#description'] = t('Links to view and log out of the current user account.');
 
+  if (module_exists('locale')) {
+    $form['components']['#options']['admin_bar.locale'] = t('Locale links');
```

Comment:
This is the "proper" name for this checkbox from a developer/technical standpoint, but I think that we should still call the user-facing label "Language switcher":
```suggestion
    $form['components']['#options']['admin_bar.locale'] = t('Language switcher');
```